_i18n/ca/_posts/2019-09-25-ERP-per-a-cooperatives.md

```diff
@@ -0,0 +1,64 @@
+
+## Proposta d’ERP: Odoo Community Edition
+
+Després d’analitzar diverses opcions de programari ERP, apostem per Odoo (anteriorment OpenERP) per dos motius principals:
+
+* Compta amb una llicència lliure LGPLv3 en la seva versió Community i molts dels seus mòduls disponibles.
+* Disposa d’una gran comunitat d’organitzacions i desenvolupadores al voltant d’Odoo que aporten millores de manera freqüent.
+
+![odoo-lpgl-agpl](/assets/post_images/2019/odoo-lpgl-agpl.png){:width="100%"}
+
+
+## Comercialització del servei ERP Odoo per cooperatives
+
+Tot i que des de Coopdevs ja tenim alguns clients fent servir Odoo a producció i alguns altres en fase de proves pilot, hem necessitat realitzar feina tècnica de preparació abans de poder-ho oferir a un número més gran de cooperatives: automatització del manteniment i gestió del servidor, actualitzacions, còpies de seguretat, monitoratge, etc.
+
+A més s’ha de tenir molt present que la part tècnica és sols una part del que cal per implantar amb èxit un ERP. Abans és necessari analitzar processos que ja tenen lloc a la cooperativa, qüestionar-los i si és necessari modificar-los abans de donar-los un suport tecnològic. La consultoria de processos de gestió i la integració d’aquests a l’ERP és una feina que implicarà temps de dedicació per part de membres de la cooperativa.  
+
+Aquesta ha estat la feina que hem fet durant el 2019 i de cara a 2020 estarem llestos per oferir el servei d’implantació d’Odoo, ofert conjuntament des de FACTO i Coopdevs, per a qualsevol cooperativa que ho sol·liciti.
+
+
+## Intercooperació en Obert
+
+Un dels temes més importants quan parlem de programari i coneixement lliures és permetre que aquest coneixement i tecnologia que posem a disposició del moviment cooperativista pugui ser utilitzat sense haver de dependre d’un proveïdor concret.
+
+L’intercooperació és cabdal pel desenvolupament del moviment cooperativista, per això proposem intercooperar també en l’àmbit ERP i la seva implantació a cooperatives per tal que la feina que es fa per determinats processos es pugui reproduir en situacions similars. Hem anomenat la intercooperació en l’àmbit de l’ERP Odoo: Odoo-Coop. Per exemple, si algú realitza implantacions d’Odoo per a botigues cooperatives, fàcilment podrem replicar les solucions per a processos similars a altres botigues cooperatives si disposem dels coneixements de les experiències anteriors.
+
+
+## Enllaços
+
+* Documentació funcional Odoo (Odoo-coop) [https://gitlab.com/odoo-coop/odoo-coop/wikis/home](https://gitlab.com/odoo-coop/odoo-coop/wikis/home) 
+* Repositori per provisionar un servidor amb Odoo [https://gitlab.com/coopdevs/odoo-provisioning](https://gitlab.com/coopdevs/odoo-provisioning) 
+* Videotutorials d’Odoo [https://www.youtube.com/playlist?list=PL1-aSABtP6AD-t0GEoxMXgCa_qLsHs5PR](https://www.youtube.com/playlist?list=PL1-aSABtP6AD-t0GEoxMXgCa_qLsHs5PR) 
+* Pàgina oficial d’Odoo Community [https://odoo-community.org/](https://odoo-community.org/) 
```

Comment:
Should we add a link to the FESC talk too? 

http://fesc.xes.cat/programa/programa-activitats-i-xerrades/ gives a 504 right now but there should be a link to our talk.